### DISCUSSION TITLE
Any suggestions to reduce ragflow image size plus memory and cpu consumption ?

### DISCUSSION THREAD

**Discussion by Greatz08** [infiniflow/ragflow#3391] [General]
its almost 9GB+ image file and when running in unrestricted mode, takes alot of ram plus i see random cpu spikes so is it possible to create lite variant of it which is working without compromises ?

**Comment by JinHai-CN** [comment]
You can try slim version docker image. RAGFlow slim Docker images do not include embedding models or Python libraries and hence are approximately 1GB in size.

Update docker/.env file: RAGFLOW_IMAGE=infiniflow/ragflow:dev-slim, then  run `docker compose` to start RAGFlow
